dataland-backend/build.gradle.kts

```diff
@@ -25,8 +27,11 @@ plugins {
 java.sourceCompatibility = JavaVersion.VERSION_17
 
 dependencies {
+    implementation(project(":dataland-backend-utils"))
     implementation(libs.springdoc.openapi.ui)
     implementation(libs.dataland.edc.client)
+    implementation(libs.moshi.kotlin)
```

Comment:
Wozu braucht man die neuen Importe?

Reply:
Habe nachgeschaut und moshi wird nur in den e2etests und im keycloak adapter genutzt.
Der keycloak adapter wiederum wird hier dependet und müsste moshi ja dann bereits mitbringen.

Daher sollte das package hier überflüssig sein.
Ich werde das jetzt nochmal validieren/ausprobieren.

Reply:
Rausgeworfen und CI ist trotzde grün.

Also fixed